src/app/story/storyrow/storyrow.component.css

```diff
@@ -46,6 +46,13 @@
     }
 }
 
+@media (min-width: 992px) {
+    .node-column {
+        /* Leave a larger gap between the main node and the siblings */
```

Comment:
This comment is conflicting with the one on #28 can you please explain why?

Reply:
The line number you mentioned applies to all screen sizes. Then this particular line makes the gap a little larger on tablets and larger devices.

Reply:
Can we please specify that in the comment?

Reply:
Addressed in the latest rebase.